src/exists.js

```diff
@@ -217,7 +217,13 @@ function testModulePath(value, fileDir) {
   return `Cannot find module '${value}'`;
 }
 
+function getConfigExtensions(context) {
+  return context.options[0] && context.options[0].extensions || [];
```

Comment:
link back/comment to the react eslint one for reference - https://github.com/yannickcr/eslint-plugin-react/blob/master/lib/rules/require-extension.js#L42